ommit 9852b8f87b28 "[net] switchdev: add VLAN support for port's
bridge_getlink" (upstream 7d4f8d871ab1), with a follow-up fix
34101a7de8599 "[net] i40e: fix call of ndo_dflt_bridge_getlink()"
(upstream 599b076d15ee3ea) changed the interface for ndo_dflt_bridge_getlink()
function.

Backport change: revert the changes.

Index: src/drivers/net/ethernet/intel/i40e/i40e_main.c
===================================================================
--- src.orig/drivers/net/ethernet/intel/i40e/i40e_main.c	2017-04-27 19:47:59.436048463 +0200
+++ src/drivers/net/ethernet/intel/i40e/i40e_main.c	2017-04-27 19:49:22.574059966 +0200
@@ -9354,7 +9354,7 @@
 		return 0;
 
 	return ndo_dflt_bridge_getlink(skb, pid, seq, dev, veb->bridge_mode,
-				       0, 0, nlflags, filter_mask, NULL);
+				       0, 0, nlflags);
 }
 
 /**
